model/storage/Storage.php

```diff
@@ -17,9 +17,9 @@
     const DIAGNOSTIC_LOGIN                = 'login';
     const DIAGNOSTIC_IP                   = 'ip';
     const DIAGNOSTIC_BROWSER              = 'browser';
-    const DIAGNOSTIC_BROWSERVERSION       = 'browserVersion';
+    const DIAGNOSTIC_BROWSERVERSION       = 'browser_version';
```

Comment:
Caution, this will be incompatible with:
- https://github.com/oat-sa/extension-tao-clientdiag/blob/fix/postgres_compatiblity/views/js/tools/browser/tester.js#L115
- https://github.com/oat-sa/extension-tao-clientdiag/blob/fix/postgres_compatiblity/views/js/tools/diagnostic/diagnostic.js#L156